types/url-search-params/index.d.ts

```diff
@@ -4,7 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare class URLSearchParams {
-    constructor(init?: string | URLSearchParams | any);
+    constructor(init?: any | URLSearchParams | string);
```

Comment:
You didn't introduce this, but it's worth fixing. `any | URLSearchParams | string` is the same exact type as `any`. Please either change this to `URLSearchParams | string` or just `any`. The current type is a bit misleading as it appears to offer a strong type guarantee but it doesn't really.